### ISSUE TITLE
sqrt doesn't render correctly when changing parent fonts

### ISSUE THREAD

**Issue by jpzwarte** [Khan/KaTeX#799]
![screenshot 2017-08-17 09 12 25](https://user-images.githubusercontent.com/2218736/29415605-8f4ee422-8331-11e7-9afe-549dcea86b09.png)
[https://www.dropbox.com/s/ax01an2q5bainve/Screenshot%202017-08-17%2009.12.25.png?dl=0]

I've noticed in my own app the the horizontal line of \sqrt doesn't always render correctly. If i change the parent font-size it sometimes disappears. You can reproduce this (in Chrome) on the KaTeX main page by disabling the font rule on the body element.

**Comment by kevinbarabash** [comment]
@jpzwarte I tried disabling the `font` rule on the body element and wasn't able to reproduce the issue.  What browser, version, and OS are you running?  We use `border-bottom-width` for the rule b/c it should (at least in theory) always be rendered even if the font is small or the user has zoomed out a lot.

**Comment by jpzwarte** [comment]
@kevinbarabash i'm running Version 62.0.3178.0 (Official Build) dev (64-bit) on a mac. My guess is that somehow on my non-retina monitor the bottom border isn't drawn because it's too small?

**Comment by jpzwarte** [comment]
I can also reproduce this on a retina display:

<img width="442" alt="screenshot 2017-08-17 13 58 41" src="https://user-images.githubusercontent.com/3968/29411230-363d4d9c-8354-11e7-8e93-00b5ac753629.png">

**Comment by jpzwarte** [comment]
Cannot reproduce on Firefox, but see a visual change happening when i disable the font rule (the horizontal line move slightly lower): 
![screenshot 2017-08-17 14 00 36](https://user-images.githubusercontent.com/3968/29411303-88fb3aee-8354-11e7-8d53-975f861560d9.png)

**Comment by jpzwarte** [comment]
@kevinbarabash does the body font-size/line-height have to be a certain value for a sqrt to render correctly?

**Comment by kevinbarabash** [comment]
It shouldn't.  The browser is supposed clamp border width at 1 (physical?) pixel, specifically so that borders don't disappear if you zoom out.  I have the exact same version of Chrome as you for Mac and can't repro it on a retina display.  I'm on 10.12.5.

**Comment by jpzwarte** [comment]
10.12.6 here but i doubt if that makes a difference.

**Comment by edemaine** [comment]
Which element is supposed to have `border-bottom-width` set?  I can't find any elements with a border, even though things render correctly for me... (on Chrome on Windows)

**Comment by kevinbarabash** [comment]
```
<span class="sqrt-line" style="border-bottom-width: 0.04em;"></span>
```

**Comment by kevinbarabash** [comment]
Fixed by #810.
